packages/toggle/README.md

```diff
@@ -0,0 +1,3 @@
+# Toggle Name
+
+Toggle documentation
```

Comment:
🔥The documentation is missing.

Reply:
@DanielMSchmidt We don't have documentation for this component, I will come up with something very simple but I would not block this PR because of the documentation.